integrations/bootstrap-vue.yml

```diff
@@ -15,122 +17,66 @@ website: 'https://bootstrap-vue.org/docs#nuxtjs-module'
 keywords:
   - a11y
   - accessibility
-  - accessibility
-  - alert
   - alert
   - aria
-  - aria
-  - bootstrap
   - bootstrap
   - bootstrap for vue
-  - bootstrap for vue
-  - bootstrap icons
   - bootstrap icons
   - bootstrap v4
-  - bootstrap v4
   - breadcrumb
-  - breadcrumb
-  - button
   - button
   - calendar
-  - calendar
-  - card
   - card
   - carousel
-  - carousel
-  - checkbox
   - checkbox
   - collapse
-  - collapse
   - components
-  - components
-  - css
   - css
   - date picker
-  - date picker
-  - datepicker
   - datepicker
   - directives
-  - directives
-  - dropdown
   - dropdown
   - dropzone
-  - dropzone
   - external
   - flexbox
-  - flexbox
-  - form
   - form
   - icons
-  - icons
-  - input
   - input
   - jquery
   - jumbotron
-  - jumbotron
-  - list
   - list
   - messagebox
-  - messagebox
   - modal
-  - modal
-  - nav
   - nav
   - navbar
-  - navbar
-  - option
   - option
   - pagination
-  - pagination
-  - polymer
   - polymer
   - popover
-  - popover
   - popper
-  - popper
-  - popper.js
   - popper.js
   - progress
-  - progress
-  - radio
   - radio
   - scss
-  - scss
-  - select
   - select
   - slider
-  - slider
   - spinner
-  - spinner
-  - ssr
   - ssr
   - tab
-  - tab
-  - table
   - table
   - tag
-  - tag
-  - tags
   - tags
   - toast
-  - toast
-  - tooltip
   - tooltip
   - uiv
   - upload
-  - upload
-  - vue
   - vue
   - vue-bootstrap
   - vue-strap
   - vue2
-  - vue2
-  - vuejs
   - vuejs
   - vuestrap
   - web
-  - web
-  - webcomponents
```

Comment:
Hmm we should do unique check with sync script isn't it working?

Reply:
not for all modules now that is why I deleted some that were double